Database_Restructure/DB_restructure_documentation.md

```diff
@@ -0,0 +1,129 @@
+**Last updated: 07/27/2022**
+
+# Database Restructuring
+
+Included files:
+- restructuring_research.xls
+- mongodb_schema_design.xls
+
+## Purpose
+
+Restructure the Postgres and MongoDB databases to be independent, clean, and comprehensible. To maintain data security, the responsibilities of the two databases are:
+
+- The Postgres database will store only authentication/authorization information in conjunction with the project's use of Okta/Auth0.
+- The MongoDB database will store all other data, i.e. all data collected from users, and backend logic will ensure that users sending or receiving that data on the frontend have the proper authorization.
```

Comment:
I've had some confusion in the last week about the relationship between Postgres and MongoDB. Thank you for making this explicitly clear in this one line--It's great to have it in writing somewhere.